process.go

```diff
@@ -16,9 +16,9 @@ package jaeger
 
 // Process holds process specific metadata that's relevant to this client.
 type Process struct {
-	Service string
-	UUID    string
-	Tags    []Tag
+	Service  string
+	ClientID string
```

Comment:
This is a breaking change. Hopefully no one has used it yet but it feels very iffy given we've already cut a version of jaeger client with UUID

Reply:
Right. I can change it back. Just changed the meaning in other places so I didn't think to leave this as UUID.

Reply:
Just change this back to uuid, the rest can still be clientID